pkg/policies/engine/rego/rego.go

```diff
@@ -145,7 +151,51 @@ func (r *Rego) Verify(ctx context.Context, policy *engine.Policy, input []byte,
 		}
 	}
 
-	return violations, nil
+	return &engine.EvaluationResult{
+		Violations: violations,
+		Skipped:    false, // best effort
+		Message:    "",
+	}, nil
+}
+
+func parseResultRule(res rego.ResultSet, policy *engine.Policy) (*engine.EvaluationResult, error) {
+	result := &engine.EvaluationResult{Violations: make([]*engine.PolicyViolation, 0)}
+	for _, exp := range res {
+		for _, val := range exp.Expressions {
+			ruleResult, ok := val.Value.(map[string]any)
+			if !ok {
+				return nil, fmt.Errorf("failed to evaluate policy evaluation result: %s", val.Text)
+			}
+
+			skipped, ok := ruleResult["skipped"].(bool)
+			if !ok {
+				return nil, fmt.Errorf("failed to evaluate 'skipped' field in policy evaluation result: %s", val.Text)
+			}
+
+			message, ok := ruleResult["message"].(string)
+			if !ok {
+				return nil, fmt.Errorf("failed to evaluate 'message' field in policy evaluation result: %s", val.Text)
+			}
+
+			violations, ok := ruleResult["violations"].([]any)
+			if !ok {
+				return nil, fmt.Errorf("failed to evaluate 'violations' field in policy evaluation result: %s", val.Text)
+			}
+
+			result.Skipped = skipped
```

Comment:
I do not think I understand this. Is this doing the `nor` of the different paths?

Reply:
This result represents a single evaluation. Merge is done in `policies` package. See `evaluatePolicyAttachment` method.